sql/distsql/base.go

```diff
@@ -71,12 +71,16 @@ type RowChannel struct {
 
 var _ rowReceiver = &RowChannel{}
 
-func (rc *RowChannel) init() {
-	rc.dataChan = make(chan StreamMsg, rowChannelBuf)
+func (rc *RowChannel) initWithBufSize(chanBuf int) {
```

Comment:
nit: naming of this parameter is slightly confusing. Why not `chanBufSize` and `rowChannelBufSize` (below)?